### PR TITLE
core:platform Add Bareos firewalld services

### PR DESCRIPTION
**Pull request by bruno-at-bareos**
This PR is about preparing official Bareos firewalld services xml before submitting them to upstream.

upstream [PR#1009](https://github.com/firewalld/firewalld/pull/1009) is now open

OP#4665

#### Please check

- [X] Short description and the purpose of this PR is present _above this paragraph_
- [X] Your name is present in the AUTHORS file (optional)

If you have any questions or problems, please give a comment in the PR.

### Helpful documentation and best practices

- [Git Workflow](https://docs.bareos.org/DeveloperGuide/gitworkflow.html)
- [Automatic Sourcecode Formatting](https://docs.bareos.org/DeveloperGuide/generaldevel.html#automatic-sourcecode-formatting)
- [Check your commit messages](https://docs.bareos.org/DeveloperGuide/gitworkflow.html#commits)

##### General
- [x] PR name is meaningful
- [x] Purpose of the PR is understood
- [x] Separate commit for this PR in the CHANGELOG.md, PR number referenced is same
- [x] Commit descriptions are understandable and well formatted
- ~~[ ] If backport: add original PR number and target branch at top of this file: **Backport of PR#000 to bareos-2x**~~

##### Source code quality

- [x] Source code changes are understandable
- [x] Variable and function names are meaningful
- [x] Code comments are correct (logically and spelling)
- [x] Required documentation changes are present and part of the PR
- [x] `bareos-check-sources --since-merge` does not report any problems
- [x] `git status` should not report modifications in the source tree after building and testing

### CODE REVIEW

File: core/platforms/firewalld/bareos-director.xml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8"?>
+<service>
+  <short>Bareos Director daemon (bareos-dir)</short>
```
Comment (joergsteffens) [inline core/platforms/firewalld/bareos-director.xml:3]:
```suggestion
  <short>Bareos Director (bareos-dir)</short>
```
In https://docs.bareos.org/DocumentationStyleGuide/CommonNames.html we define a common writing style for the daemon.
Still, there are places where the naming has been "Bareos Director Daemon" /with captital D). I'd also agree to tihs.

**Review by joergsteffens — CHANGES_REQUESTED** [review]

File: core/platforms/firewalld/bareos-director.xml (hunk 1 above)
Comment (bruno-at-bareos) [reply core/platforms/firewalld/bareos-director.xml:3]:
```suggestion
  <short>Bareos Director Daemon (bareos-dir)</short>
```

Comment (bruno-at-bareos) [reply core/platforms/firewalld/bareos-director.xml:3]: I will prefer to have the 3 named Daemon.

File: core/platforms/firewalld/README.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
+# Bareos firewalld service files
+
+The files bareos-director.xml bareos-storagedaemon.xml bareos-filedaemon.xml
```
Comment (joergsteffens) [inline core/platforms/firewalld/README.md:3]:
```suggestion
The files bareos-director.xml bareos-storage.xml bareos-filedaemon.xml
```

**Review by joergsteffens — APPROVED** [review]

**Pull request merged**